libtransmission/variant.h

```diff
@@ -180,7 +180,7 @@ struct tr_variant
     tr_variant& operator=(tr_variant&& that) noexcept = default;
 
     template<typename Val>
-    tr_variant(Val value)
+    tr_variant(Val&& value)
     {
         *this = std::move(value);
```

Comment:
This will need to be changed as well then,
```suggestion
        *this = std::forward(value);
```